packages/amplify-category-custom/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "amplify-cli-core": "3.2.2",
     "amplify-prompts": "2.6.0",
+    "aws-cdk-lib": "^2.47.0",
```

Comment:
I'd keep it at `^2.41.0` to match data and other places. We should bump it later.